src/services/accommodation.service.js

```diff
@@ -42,6 +42,45 @@ class AccommodationService {
       where: { ...property }
     });
   }
+
+  /**
+ * find Accommodation Booking
+ * @static
+ * @param {object} property
+ * @memberof AccommodationService
+ * @returns {object} data
+ */
+  static findBookingByProperty(property) {
```

Comment:
@higustave-ops Any reason why this PR contains changes which are not relevant to its story?